docs/lessons/dance-party.md

```diff
@@ -0,0 +1,511 @@
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+`, true)
+scene.setTileMap(img`
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    1 1 1 1 1 1 1 1 1 1
+`)
+effects.starField.startScreenEffect()
+```
+
+Congratulations! You have now set the scene for your game.
+
+## Part Two: Creating the Sprite
+
+To set the sprite grab a ``||sprites:set mySprite to sprite of kind player||`` from ``||sprites:sprites||`` and drag it into ``||loops:on start||``. Now click on mySprite and select rename variable and rename your sprite to whatever you would like to call your player. In the ``||sprites:sprite of kind player||`` block, select the grey square and draw an image of your player. 
+
+Now we want to set our sprites starting position. To do this go into sprites and grab ``||sprites:set mySprite position to x y||`` and drag it under ``||sprites:sprite of kind player||``. Now set the x position to **80** and the y position to **100**. 
+
+```blocks
+let steve: Sprite = sprites.create(img`
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . f f f . . . . . . .
+    . . . . . . f f f . . . . . . .
+    . . . . . . f f f . . . . . . .
+    . . . . . d 9 9 9 d . . . . . .
+    . . . . . d 9 9 9 d . . . . . .
+    . . . . . d 8 8 8 d . . . . . .
+    . . . . . d 8 8 8 d . . . . . .
+    . . . . . . 8 . 8 . . . . . . .
+    . . . . . . 8 . 8 . . . . . . .
+    . . . . . . c . c . . . . . . .
+    . . . . . . . . . . . . . . . .
+`, SpriteKind.Player)
+steve.setPosition(80, 100)
+```
+
+
+## Part Three: Move the Player
+
+Our next step is to create a player that can move strictly between four spots. To do this go to ``||controller:controller||`` and grab ``||controller:on button pressed||`` and drag it into your workspace. Then select the first drop down and select **right**. Now grab a ``||sprites:set mySprite position to x y||`` and under the drop down select your sprite. For the x value type in **130** and then for the y value type in **100**. 
+
+Select the ``||controller:on button pressed||`` containing the ``||sprites:set mySprite position to x y||`` and duplicate it 3 times. Now set the new copies to up, down, and left. Because we want the y variable to remain the same, only change the x positions. 
```

Comment:
wrap all the places where you're talking about variables / properties / etc in backticks so they show up differently; they're inherently short pieces of code so it helps break up things people should be looking for from things people should be reading exactly. e.g. here: x, y, up, down, left